### DISCUSSION TITLE
Invalidate cache when working on the endpoint

### DISCUSSION THREAD

**Discussion by Barbapapazes** [nitrojs/nitro#1869] [Q&A]
Hello,

Once a cache route rules is set on an endpoint, returned content is cached using file system. But, if you work on the endpoint, changing code, updated the returned content, the cache is still used which is annoying. Couldn't be possible to invalidate (or remove the cached content) when the endpoint is saved?

**Comment by pi0** [comment]
By default, we use the integrity of the cached handler and invalidate cache entries when the (handler code) changes. You can also manually provide a custom integrity or use `$development: { routeRules: { '/path': { cache: false } }` or a similar way to disable caching conditionally only for development time.
